pyprof/prof/linear.py

```diff
@@ -64,10 +67,13 @@ def setXWBMNK(self, args):
         n, k2 = w['shape']
         assert (k1 == k2)
         if b is not None:
-            assert (b['shape'][0] == n)
+            # assert (b['shape'][0] == n)
+            assert(b['shape'][0] == n or 
+                   (b['shape'][0] == 1 and b['shape'][1] == n))
         t1 = x['dtype']
         t2 = w['dtype']
-        assert (t1 == t2)
+        # no longer true
+        # assert (t1 == t2)
```

Comment:
Please provide a test case.

Reply:
Do not leave commented code. Please delete if no longer needed.